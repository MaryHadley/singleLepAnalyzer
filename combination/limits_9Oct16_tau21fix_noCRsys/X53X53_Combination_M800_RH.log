000 (1297374.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297374.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.239:46789>
...
006 (1297374.000.000) 10/10 19:27:27 Image size of job updated: 188468
	83  -  MemoryUsage of job (MB)
	84116  -  ResidentSetSize of job (KB)
...
006 (1297374.000.000) 10/10 19:32:28 Image size of job updated: 283216
	159  -  MemoryUsage of job (MB)
	161996  -  ResidentSetSize of job (KB)
...
006 (1297374.000.000) 10/10 20:17:32 Image size of job updated: 284276
	160  -  MemoryUsage of job (MB)
	163000  -  ResidentSetSize of job (KB)
...
005 (1297374.000.000) 10/10 20:20:53 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:53:12, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:53:12, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1629  -  Run Bytes Sent By Job
	322  -  Run Bytes Received By Job
	1629  -  Total Bytes Sent By Job
	322  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1100        1   3824502
	   Memory (MB)          :      160     3072      3072
...
