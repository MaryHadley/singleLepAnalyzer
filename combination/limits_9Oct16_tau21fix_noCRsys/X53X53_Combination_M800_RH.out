/var/lib/condor/execute/dir_12461
Total number of likelihood evaluations: 55000100
Total number of likelihood evaluations: 5500100
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M800_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M800_RH/quant-toys:0-sig-2106c0539d.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M800_RH/quant-data-sig-1743beb2fb.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M800_RH
