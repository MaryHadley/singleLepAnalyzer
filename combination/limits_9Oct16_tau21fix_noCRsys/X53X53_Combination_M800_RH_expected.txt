# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.085401   0.043152    0.18069  0.058794    0.12794
